### PR TITLE
Feature/df 794 stelle beanstanden

### PR DESCRIPTION
**Pull request by fabiangotzen**
specification: https://alv-ch.atlassian.net/wiki/spaces/OS/pages/286556237/UI+Stelle+beanstanden
(Modal inside modal was specified) 

backend: https://github.com/alv-ch/job-ad-service/pull/320

IE: 
image

mobile: 
image

### CODE REVIEW

File: alv-portal-ui/translations.csv [hunk 1]
```diff
@@ -504,6 +504,7 @@ home.tools.jobPublicationTab,Stelle melden,Post job,Publier un poste vacant,Annu
 job-detail.action.applyNow,Jetzt bewerben,Apply now,Postuler maintenant,Candidarsi ora,
 job-detail.action.copyLink,Link kopieren,Copy link,Copier le lien,Copiare il link,
 job-detail.action.print,Drucken,Print,Imprimer,Stampare,
+job-detail.action.complain,Stelle beanstanden,,,,
```
Comment (emina-str) [inline alv-portal-ui/translations.csv]: @fabiangotzen  can you please add all the missing translations to ticket https://alv-ch.atlassian.net/browse/DF-808

Comment (fabiangotzen) [reply alv-portal-ui/translations.csv]: Yes, I will add them. Thanks

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/complaint-modal/complaint-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  ngOnInit() {
+    this.form = this.fb.group({
+      salutation: [null, Validators.required],
+      name: ['', [Validators.required, Validators.maxLength(this.MAX_LENGTH_255)]],
+      phone: [''],
+      email: ['', [Validators.required, Validators.maxLength(this.MAX_LENGTH_255)]],
+      complaintMessage: ['', [Validators.required, Validators.maxLength(this.MAX_LENGTH_1000)]]
+    });
+
+    combineLatest(this.authenticationService.getCurrentCompany()).pipe(
```
[the remark this answers is not in the document]
Comment (fabiangotzen) [reply alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/complaint-modal/complaint-modal.component.ts]: removed, thanks

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/job-detail.component.html [hunk 3]
```diff
@@ -158,6 +162,14 @@
             {{'portal.job-detail.sendLink' | translate}}
           </a>
         </li>
+        <li class="d-none d-md-block">
+          <button class="btn btn-text btn-text-icon btn-truncate"
+                  (click)="openComplaintModal()">
```
Comment (emina-str) [inline alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/job-detail.component.html]: are you missing an argument here?

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/complaint-modal/complaint-modal.component.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        if (!!templateInfo) {
+          this.patchTemplateValues(templateInfo);
+        }
+      });
+  }
+
+  onSubmit(form: FormGroup) {
+    const formValue = <ComplaintFormValue>form.value;
+    this.complaintRepository.sendComplaint(mapFormToDto(this.jobAdvertisementId, formValue))
+      .pipe()
```
Comment (99999989) [inline alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/complaint-modal/complaint-modal.component.ts]: Remove empty call of `pipe()`

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/job-detail.component.html (hunk 3 above)
Comment (fabiangotzen) [reply alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/job-detail.component.html]: Yes! :) fixed.

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/complaint-modal/complaint-modal.component.ts (hunk 4 above)
Comment (fabiangotzen) [reply alv-portal-ui/src/app/job-advertisement/job-ad-search/job-detail/complaint-modal/complaint-modal.component.ts]: removed

**Pull request merged**